examples/Example01_Basic_OneShot/Example01_Basic_OneShot.ino

```diff
@@ -43,36 +45,35 @@ void setup() {
   Serial.println("Sensor began.");
```

Comment:
Couple comments on `myDevice.begin()` in general:

I've always found it much more helpful to have a look that continuously attempts to begin the device (1x per second is fine, don't go crazy), and prints an error message each time. I know, you shouldn't hotplug devices, but people are going to do it anyways, and this just saves them a step of having to hit the reset button (or re-uploading code if they don't have a reset button (yes, really, some dev boards don't)). And then there's much less chance of the user missing the single error message if they didn't have their serial monitor open or something.

If you insist on freezing, then change "Spinning" to "Freezing" or "Halting" or something that's more obvious (think about non-native English speakers).

Reply:
I do insist on freezing, it's either going to be connected or it won't. I don't want to encourage bad behavior. I'll rename it to halting.